configuration/app-settings.md

```diff
@@ -11,9 +11,96 @@ The following settings do not need to be specified. They should only be defined
+
+### Configuring SMS schedules
+
+The Medic platform can have a set of predetermined messages set to be sent at a specificied times in future. These messages are usually
+triggered by form submissions. To configure scheduled messages, the following declarations must be made in app_settings.json:
+
+#### 1. Set up the scheduled messages
+
+Scheduled messages are defined under the `schedules` key as an array of schedule objects. The definition takes the typical form below:
+
+```
+  "schedules": [
+    {
+      "name": "ANC Visit Reminders",
+      "summary": "",
+      "description": "",
+      "start_from": "lmp_date",
+      "messages": [
+        {
+          "message": [
+            {
+              "content": "Please remind Jane to visit the health facility for ANC visit this week. Thanks!",
+              "locale": "en"
+            },
+            {
+              "content": "Tafadhali mkumbushe Jane ahudhurie kliniki kwa ANC wiki hii. Akishafanya hivyo tujulishe kwa kutuma ripoti ya kuhakiki kliniki ya ANC. Asante!",
+              "locale": "sw"
+            }
+          ],
+          "group": 1,
+          "offset": "4 weeks",
+          "send_day": "monday",
+          "send_time": "09:00",
+          "recipient": "reporting_unit"
+        },
+		.
+		.
+		.
+		.
+		more message definitions
+      ]
+    }
+	.
+	.
+	.
+	.
+	more schedule definitions
+]
+```
+
+|Setting|Mandatory|Description|
+|-------|---------|----------|
+|name|yes|A unique string label that is used to identify the schedule. Spaces are allowed.|
+|summary|no|Short description of the of the schedule.|
+|description|no|A narrative for the schedule.|
+|start_from|no|A date field from which exact scheduled message dates will be calculated. If not specified, the reported date will be used instead. A field from the report is usually used to determine when scheduled messages will go out.|
+|messages|yes|This is an array of settings for each message going out.<ul><li>"message": These are the actual messages set to go out in future. To achieve multiple language support, they can be set as an array of content locale pairs as in the example above, or if using version 2.15+, translation strings can be used as the message text.</li><li>"group": An integer used to categorize messages when displayed on the webapp.</li><li>"offset": A time interval from the "start_from" date that defines when the message should be sent. It can be in days or weeks.</li><li>"send_day": Day of the week the message should be sent.</li><li>"send_time": Time of day that the message should be sent in 24 hour format.</li><li>"recipient": Recipient of the message. It can be set to:<br/>'reporting_unit' - sender of the form<br/>'clinic' - clinic that the sender of the form is attached to<br/>'parent' - parent of the sender of the form.</li></ul>|
```

Comment:
I updated the text, and broken down further so that each row is a (sub)property, as seen here: https://github.com/medic/medic-docs/blob/9ff45ff4b9124fb4ff05a2fc7a67b5c6b783c74f/configuration/developing-community-health-applications.md#task-properties

```suggestion
|`messages`|Array of objects, each containing a message to send out and its properties.|yes|
|`messages[].translation_key`| The translation key of the message to send out. Available in 2.15+.|yes|
|`messages[].messages`| Array of message objects, each with `content` and `locale` properties. From 2.15 on use `translation_key` instead.|no|
|`messages[].group`|Integer identifier to group messages that belong together so that they can be cleared together as a group by future reports. For instance a series of messages announcing a visit, and following up for a missed visit could be grouped together and cleared by a single visit report. |yes|
|`messages[].offset`| Time interval from the `start_from` date for when the message should be sent. It is structured as a string with an integer value followed by a space and the time unit. For instance `8 weeks` or `2 days`. The units available are `seconds`, `minutes`, `hours`, `days`, `weeks`, `months`, `years`, and their singular forms as well. Note that although you can specify `seconds`, the accuracy of the sending time will be determined by delays in the processing the message on the server and on the gateway.|yes|
|`messages[].send_day`| String value of the day of the week on which the message should be sent. For instance, to send a message at the beginning of the week setting it to `"Monday"` will make sure the message goes out on the closest Monday _after_ the `start_date` + `offset`. `|no|
|`messages[].send_time`| Time of day that the message should be sent in 24 hour format.|no|
|`messages[].recipient`| Recipient of the message. It can be set to `reporting_unit` (sender of the form), 'clinic' (clinic that the sender of the form is attached to), 'parent' (parent of the sender of the form), or a specific phone number.|no|
```